cloudant/v1.ts

```diff
@@ -14,7 +14,7 @@
  * limitations under the License.
  */
 
-import * as extend from 'extend';
+import extend from 'extend';
```

Comment:
This issue would have been visible at compile time instead of runtime if we had `@types/extend` in the `devDepedencies`. Perhaps something to consider adding.

Reply:
sure, done in https://github.com/IBM/cloudant-node-sdk/pull/1416/commits/0ba57fdcdf8b79a2a40fa1e673d24efba1c71b9f